src/relay/op/tensor/transform.cc

```diff
@@ -777,6 +787,13 @@ bool FullLikeRel(const Array<Type>& types,
   return true;
 }
 
+Array<Tensor> FullLikeCompute(const Attrs& attrs,
+                              const Array<Tensor>& inputs,
+                              const Type& out_type,
+                              const Target& target) {
+  return { topi::full_like(inputs[0], inputs[1]()) };
```

Comment:
If I am you, i will CHECK inputs size before return.
Maybe we should have a convention for how much CHECKing we should do? idk.

Reply:
input size is verified at the FullLikeRel, I guess again checking will b reduntant. most of new compute func doesnt have this checks.